samples/crypto/aes_ccm/README.rst

```diff
@@ -7,24 +7,28 @@ Crypto: AES CCM
    :local:
    :depth: 2
 
+The AES CCM sample shows how to perform AES authenticated encryption and decryption operations using the CCM block cipher mode and a 128-bit AES key.
```

Comment:
Possible redundancy to state "block cipher" behind CCM as CCM is an acronym for "Counter Mode with Cipher Block Chaining (CBC) Message Authentication Code (MAC)" (aggregate of CBC and MAC)

I would call it the CCM algorithm in this context as the rest is explained (AEAD)

Reply:
So are you proposing something like the following one?

The AES CCM sample shows how to perform authenticated encryption and decryption operations using the CCM algorithm and a 128-bit key.

Reply:
Yes. The only thing we are missing is to mention the AEAD directly, but I didn't want to make this more complex than it is. Your term is good